de_web_plugin.cpp

```diff
@@ -5718,34 +5747,99 @@ void DeRestPluginPrivate::updateSensorNode(const deCONZ::NodeEvent &event)
                     {
                         for (;ia != enda; ++ia)
                         {
-                            if (ia->id() == 0x0000) // temperature (0.01 °C)
+                            if (i->modelId().startsWith(QLatin1String("VOC_Sensor")))
```

Comment:
Same suggestion as above, currently the code of the temperature attribute 0x0000 is duplicated.
I think this can be simplified by just adding the special humidity attribute 0x0001 handling code in an else if (), so that the VOC_Sensor can use the common temperature cluster 0x0000 attribute code.

```c++
if (ia->id() == 0x0000) // temperature
{
   // ...
}
else if (i->modelId().startsWith(QLatin1String("VOC_Sensor")) && ia->id() == 0x0001)
{
    // humidity sensor values are transferred via temperature cluster 0x0001 attribute
    // see: https://github.com/dresden-elektronik/deconz-rest-plugin/pull/1964
   
   // ...
}
```

Reply:
Hello @manup!
I've changed code according to your suggestions.
Could you please review it.